src/runtime/Makefile

```diff
@@ -186,7 +183,7 @@ DEFENTROPYSOURCE := /dev/urandom
 DEFDISABLEBLOCK := false
 DEFSHAREDFS := virtio-9p
 DEFSHAREDFS_QEMU_VIRTIOFS := virtio-fs
-DEFVIRTIOFSDAEMON := $(VIRTIOFSDBINDIR)/virtiofsd
+DEFVIRTIOFSDAEMON := $(LIBEXECDIR)/kata-qemu/virtiofsd
```

Comment:
@bergwolf, what's the reason of adding `kata-qemu` to the path?

Using $(LIBEXECDIR) would be enough and even help distros to not have to `DEFVIRTIOFSDAEMON` when building the project.

Reply:
This is becase we are building and shipping `kata-static` package by default. It puts all kata built qemu staff in a separate `kata-qemu` directory to avoid conflicts with distro qemu packages.

Reply:
Okay, I'm not sure whether I agree 100% with the approach, but I don't feel any need to disagree on this now. :-).

Later on, **after the release**, we can discuss whether it's the best to use the defaults on our Makefile to match what would be used in a distro or to match what's shipped with kata-deploy.

Thanks for the answer, I'm approving the PR!